src/System.Private.CoreLib/shared/System/Convert.cs

```diff
@@ -2492,19 +2494,146 @@ public static unsafe bool TryToBase64Chars(ReadOnlySpan<byte> bytes, Span<char>
             }
         }
 
+        internal static readonly Vector128<byte> s_base64ShuffleMask = Vector128.Create((byte)
```

Comment:
A short comment describing each constant would be useful.

It's also not clear why these are static readonly, but several of the others (such as `tt0-tt8`) are not

Reply:
Given https://github.com/dotnet/coreclr/issues/17225 and https://github.com/dotnet/coreclr/issues/26976, it would be more efficient processing and space-wise to use the `ROS<byte>` read-only property trick on these, especially since they're only used by code behind a `Ssse3.IsSupported` check.